### PR TITLE
Ukrainian locale fixes

### PR DESCRIPTION
**Pull request by v-atamanenko**

### CODE REVIEW

**Comment by v-atamanenko** [comment]
[Uploadcare Widget: Ukrainian locale](https://trello.com/c/kek8eP9l/142-uploadcare-widget-ukrainian-locale)

File: app/assets/javascripts/uploadcare/locale/uk.js.coffee [hunk 1]
```diff
@@ -142,10 +142,10 @@ uploadcare.namespace 'locale.translations', (ns) ->
 # http://unicode.org/repos/cldr-tmp/trunk/diff/supplemental/language_plural_rules.html
 
 uploadcare.namespace 'locale.pluralize', (ns) ->
```
Comment (bautrukevich) [inline app/assets/javascripts/uploadcare/locale/uk.js.coffee]: @v-atamanenko how about pluralization?

Comment (v-atamanenko) [reply app/assets/javascripts/uploadcare/locale/uk.js.coffee]: It is done right. I've re-checked.

**Comment by Ne4istb** [comment]
@v-atamanenko Thanks for taking care of proper translations.

**Pull request merged**